2018-03-28/azblob/highlevel.go

```diff
@@ -372,7 +372,8 @@ func UploadStreamToBlockBlob(ctx context.Context, reader io.Reader, blockBlobURL
 	result, err := uploadStream(ctx, reader,
 		UploadStreamOptions{BufferSize: o.BufferSize, MaxBuffers: o.MaxBuffers},
 		&uploadStreamToBlockBlobOptions{b: blockBlobURL, o: o, blockIDPrefix: newUUID()})
-	return result.(CommonResponse), err
+	commonResponse, _ := result.(CommonResponse)
```

Comment:
Could you please rebase on top on the dev branch? A bit of code has changed here.

Reply:
OK